File: Maximum Gap (Array Series #4).ts
export function maxGap(numbers: number[]): number {
  numbers.sort((a,b) => a-b)
  let max = -1
  for(let i=1;i<numbers.length;i++){
    if(numbers[i]-numbers[i-1]>max){
      max = numbers[i]-numbers[i-1]
    }
  }
  return max
}

File: Maximum Gap (Array Series #4)/Maximum Gap (Array Series #4).dart
int maxGap(nums) {
  nums.sort();
  int max = -1;
  for(var i=1;i<nums.length;i++){
    if(nums[i]-nums[i-1]>max){
      max = nums[i]-nums[i-1];
    }
  }
  return max;
}
